OmniKit/Model/FaultEventCode.swift

```diff
@@ -17,12 +17,15 @@ public struct FaultEventCode: CustomStringConvertible {
         case failedFlashErase                     = 0x01
         case failedFlashStore                     = 0x02
         case tableCorruptionBasalExtraCommand     = 0x03
+        case faultEventType04                     = 0x04
```

Comment:
FaultEventCode type already handles unknown types, that will stringify as "Fault Event Code %1$02x".  We shouldn't add any enums for types that we don't know anything about.

Reply:
Good idea, I was also a bit in doubt if this would be a good idea to add. I will remove the unknown ones.